src/pat-sortable-table.js

```diff
@@ -13,6 +14,8 @@ parser.addArgument("paging-type", "numbers", [
 ]);
 parser.addArgument("page-length", 20);
 parser.addArgument("page-length-menu", [10, 20, 50, 100]);
+parser.addArgument("server-side", false);
+parser.addArgument("ajax", "");
```

Comment:
Do we need the ``server-side`` option if we have ``ajax``? Cornelis is a bit averse to booleans as CSS‌ does not know boolean values.